torch_xla/debug/profiler.py

```diff
@@ -68,3 +80,59 @@ def trace(service_addr: str,
       duration_ms=duration_ms,
       num_tracing_attempts=num_tracing_attempts,
       options=options)
+
+
+class Trace(torch_xla._XLAC.profiler.TraceMe):
```

Comment:
perhaps beyond the scope of this pr, but we may think about having an env var, that controls the number of steps to profile for. What happens if I run the test_train script above for too long? Are we relying on the user ctrl-c'ing to stop profiling? WDYT about raising an error in the __enter__ or __exit__ if the metric counter for `MarkStep` is greater than or equal to some value?